src/main/java/ontos/ui/Ui.java

```diff
@@ -0,0 +1,231 @@
+package ontos.ui;
+
+import java.util.Scanner;
+
+import ontos.task.Task;
+import ontos.task.TaskList;
+
+/**
+ * The Ui class handles the interaction with the user by reading input
+ * and displaying relevant messages. It includes methods to greet the user,
+ * display tasks, handle incorrect input, and various task-related actions.
+ */
+public class Ui {
+    /** Greeting message displayed when the application starts. */
+    private static String hello = " Hello! I'm Ontos \n What can I do for you?";
```

Comment:
Should all the messages be declared as private static final like constants and named in all caps with a common prefix like MESSAGE_HELLO?